src/components/objectListing/objectListing.component.tsx

```diff
@@ -331,13 +334,35 @@ export const ObjectList = ({
   });
   const { virtualItems: virtualRows, totalSize } = rowVirtualizer;
 
+  console.log("rowVirtualizer", rowVirtualizer);
+
+  /*
+  const primaryKey = [
+    config.primaryField || "",
+    ...DISPLAY_NAME_PRIORITY,
+  ].find((field) => !!object[field]);
+*/
   return (
     <div
       className={clsx(
         "flex h-full flex-col space-y-4",
         isPanelOpen ? "lg:space-y-8" : "md:space-y-8",
       )}
     >
+      <DragOverlay modifiers={[snapCenterToCursor]}>
+        {draggedObject ? (
+          <div className="my-o flex max-w-[350px] items-center space-x-2 ">
+            <Pill
+              label={draggedObject.object.__typename as string}
+              bgColor={draggedObject.config.colour}
+              className="w-20"
+            />
+            <div className="flex flex-1">
+              <p>{draggedObject.object.title || draggedObject.object.uid}</p>
```

Comment:
this should be using the `primaryKey` bit that is commented out above

Reply:
oh yes 🤦 forgot to implement, but thought about that 😉